data/pigui/libs/wrappers.lua

```diff
@@ -744,4 +746,31 @@ function ui.maybeSetTooltip(tooltip)
 	end
 end
 
+--
+-- Function: ui.incrementDrag
+--
+-- ui.incrementDrag(label, value, v_min, v_max, format)
+--
+-- Create a "drag with arrows and progress bar" widget
+--
+-- Example:
+--
+-- > value = ui.incrementDrag("##mydrag", value, 0, 20, "%dt")
+--
+-- Parameters:
+--
+--   label - string, text, also used as ID
+--   value - int, set drag to this value
+--   v_min - int, lower bound
+--   v_max - int, upper bound
+--   format - optional string, format according to snprintf
+--
+-- Returns:
+--
+--   value - the value that the drag was set to
+--
+function ui.incrementDrag(label, value, v_min, v_max, format)
+	return pigui.IncrementDrag(label, value, v_min, v_max, defaultTheme.styleColors.accent_500, format)
+end
```

Comment:
Instead of passing a hardcoded color variable, please use `ImGui::GetColorU32` in the C++ widget (with the appropriate ImGui style color) for consistency with the rest of the ImGui widgets - this allows the theme to simply push `FrameBgActive = styleColors.accent_500` instead of having to update every call point. This would also allow the widget definition to be moved to `forwarded.lua` instead of in the wrappers file.

Reply:
Done!